src/main/resources/org/apache/camel/language/datasonnet/datasonnet.json

```diff
@@ -7,13 +7,13 @@
     "description": "To use Datasonnet scripts in Camel expressions or predicates.",
     "deprecated": false,
     "deprecationNote": "language,datasonnet",
-    "firstVersion": "1.0.0",
+    "firstVersion": "3.3.0",
```

Comment:
Shouldn't this be 3.0.1? Though I guess it doesn't matter much until we submit, and will probably be 3.3.0 then

Reply:
I think this is a file that's only needed for camel components that are bundled in the camel project. So I'm not sure if we should maintain it manually :thinking: it'll eventually be whatever camel version is current if/when it is adopted.

Reply:
Yeah, for sure. I'm good with it moving to 3.3.0 I think

Reply:
Presumably we'll need to tick it to the first version it's actually included with or something like that

Reply:
Yeah I think so